data/en/nowserver.json

```diff
@@ -4,7 +4,7 @@
 	"syntax": "nowServer()",
 	"returns": "date",
 	"related": [],
-	"description": "",
+	"description": "This function is deprecated. Returns the current time on the server independent of Lucee timezone definition",
```

Comment:
Move "This function is deprecated." from `description` to `engines.lucee.deprecated`.